sources/core/input_reader.py

```diff
@@ -11,7 +11,14 @@ def read_encrypted_base64_text():
         while True:
             line = input()
 
-            if line == Formatter.HEADER:
+            if line[:39] == Formatter.HEADER:
```

Comment:
**Magic numbers are prohibited.** In addition if you use it multiple times please use a constant.

What's 39? The length of the header? If yes, we could calculate it. If it's too heavy we could save as recalculated value during a session.